deploy-edpm.yml

```diff
@@ -117,6 +117,13 @@
         mode: "0644"
 
 - name: Run log related tasks
```

Comment:
it can be merged in previous play execution :) 
In other words, remove L119-L122